trieste/acquisition/rule.py

```diff
@@ -1484,122 +1525,81 @@ def filter_datasets(
         return filtered_datasets
 
 
-class UpdatableTrustRegionBox(Box, UpdatableTrustRegion):
-    """
-    A simple updatable box search space with a centre location and an associated global search
-    space.
-    """
+# TODO: what is a good name for this class?
+class GenericUpdatableTrustRegion(UpdatableTrustRegion):
```

Comment:
why exactly cant we combine this one with `UpdatableTrustRegion`? why do we need separate classes?

Reply:
ok, re-read the thread above, so I think now understand it a bit better

Reply:
This is a specific algorithm for intialize/update. For example, TURBO is different and doesn't use this class.